src/kernel/cuda/binary_reduce_impl.cuh

```diff
@@ -98,13 +145,14 @@ struct BinaryReduceBcast {
     DType* lhsoff = gdata->lhs_data + lid * gdata->lhs_len;
     DType* rhsoff = gdata->rhs_data + rid * gdata->rhs_len;
     DType* outoff = gdata->out_data + oid * gdata->out_len;
-    int64_t tmp[NDim];  // store unraveled idx.
```

Comment:
Does this mean `NDim` template argument is not needed any more?

Reply:
> Does this mean `NDim` template argument is not needed any more?

Yes, it is not needed in this func. Maybe further opt may need this again.

Reply:
Could you remove the argument? It will make the compilation faster.

Reply:
> Could you remove the argument? It will make the compilation faster.

Oops, NDim is not need by the code, but it is needed by one of the arguments of ApplyEdge func: static __device__ __forceinline__ void ApplyEdge(Idx src, Idx dst, Idx eid, **BcastGData<NDim, Idx, DType>* gdata**) {